packages/fluentui/CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### BREAKING CHANGES
 - Styles from `ChatMessage` content slot were moved to exact component `ChatMessageContent` @yuanboxue-amber ([#24691](https://github.com/microsoft/fluentui/pull/24691))
 - Remove `goToToday` prop from `DatePicker` as it had no effect @miroslavstastny ([#25242](https://github.com/microsoft/fluentui/pull/25242))
+- `nonce` can be configured for Fela renderer @layershifter ([#25992](https://github.com/microsoft/fluentui/pull/25992))
```

Comment:
nit: "`createFelaRenderer` is now a function" for a breaking change, nonce as a feature